app/Models/Grades.php

```diff
@@ -8,7 +8,7 @@ class Grades extends Model
 {
     protected $tables = 'grades';
     protected $fillables = [
-    	'id',
-    	'name'
+        'id',
+        'name'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Models/Grades.php</br>**Line**: 12</br>Comma required after last value in array declaration